reactive/src/main/java/feign/reactive/ReactorFeign.java

```diff
@@ -30,7 +30,7 @@ public static Builder builder() {
 
   public static class Builder extends ReactiveFeign.Builder {
 
-    private Scheduler scheduler = Schedulers.elastic();
```

Comment:
Problem is this breaks binary compatibility.... is so annoying when projects break compatibility on minor releases

Reply:
can we make `Scheduler scheduler` a parameter, and if never set, we default to `Schedulers.boundedElastic()`?

And then document how to use 3.4

Reply:
Just for my understanding: Why is it breaking binary compatibility?

It appears to me that the <code>scheduler</code> is already a parameter, which you can set with <code>public Builder scheduleOn(Scheduler scheduler)</code> and <code>Schedulers.elastic()</code> is the current default.

Reply:
`elastic` to `boundedElastic` breaks anyone using the old version of the library.

Tell you what, as a compromise, move the initialization to constructor and have one that allows user to pick how to create the scheduler and the other defaults to new code

```
  public static Builder builder() {
    return new Builder();
  }

  public static Builder builder(Scheduler scheduler) {
    return new Builder(scheduler);
  }

  public static class Builder extends ReactiveFeign.Builder {

    private Scheduler scheduler;

.. constructors...
```